uvm/servlets/admin/config/network/view/Advanced.js

```diff
@@ -1440,8 +1448,11 @@ Ext.define('Ung.config.network.view.Advanced', {
                                     bind: '{record.virtualLinks.list}',
                                     maxHeight: 140,
                                     emptyRow: {
-                                        field1: ''
+                                        field1: ''.t(),
```

Comment:
This is unnecessary; we don't do translations on empty strings.

Reply:
Oh Sorry, typo, will change it